proto/panacea/oracle/v2/query.proto

```diff
@@ -24,7 +24,7 @@ service Query {
 
   // OracleRegistrations returns a list of OracleRegistration.
   rpc OracleRegistrations(QueryOracleRegistrationsRequest) returns (QueryOracleRegistrationsResponse) {
-    option (google.api.http).get = "/panacea/oracle/v2/oracle_registrations";
+    option (google.api.http).get = "/panacea/oracle/v2/oracle_registrations/{unique_id}";
```

Comment:
```suggestion
    option (google.api.http).get = "/panacea/oracle/v2/oracle-registrations/{unique_id}";
```
Let's go with hyphens instead of underscores: https://developers.google.com/search/docs/crawling-indexing/url-structure?hl=en&visit_id=638059984940645869-1797957887&rd=1